File: Lab2KarpaBohdan36/src/Lab2KarpaBohdan36/Stem.java
package Lab2KarpaBohdan36;

/**
 * Клас, що представляє стебло рослини.
 */
public class Stem {
    private double height;

    public Stem(double height) {
        this.height = height;
    }

    public double getHeight() {
        return height;
    }

    public void setHeight(double height) {
        this.height = height;
    }

    @Override
    public String toString() {
        return "Висота стебла: " + height + " см";
    }
}

File: plants/stem.py
class Stem:
    """Клас, що представляє стебло рослини."""

    def __init__(self, height):
        """Ініціалізує висоту стебла."""
        self.height = height

    def get_height(self):
        """Повертає висоту стебла."""
        return self.height

    def set_height(self, height):
        """Задає нову висоту стебла."""
        self.height = height

    def __str__(self):
        return f"Висота стебла: {self.height} см"
